app/main.py

```diff
@@ -2,0 +1,19 @@
+from functools import reduce
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(players: list[Player]) -> int:
+    return reduce(lambda accum,
+        player: accum + player.get_rating(), players, 0)
+
+
+def elves_concert(elves: list[Elf]) -> None:
+    for elf in elves:
+        elf.play_elf_song()
```

Comment:
The `elves_concert` function correctly iterates over the list of elves and calls the `play_elf_song` method. Ensure that all elf classes have this method implemented.